tests/integration.rs

```diff
@@ -1338,12 +1336,10 @@ async fn projection_state(
     let events = generate_events("testing", 10);
     let stream_name = gen_name.next().unwrap();
 
-    let write_result = stream_client
+    stream_client
```

Comment:
Worth testing if `write_result` has a sensible `next_expected_version` and `position`?

Reply:
I second that suggestion 👍

Reply:
The old code made sure that the write request went well. It's a test related to `projections` where the actual `WriteResult` has no purpose.

Flattening the result we got out of `append_to_steam` means we have the same behavior as before after the `.await?` call.

However, I added a meaningful `WriteResult` check somewhere it's needed.

Reply:
https://github.com/EventStore/EventStoreDB-Client-Rust/pull/116/files#diff-eb771f54a1fdc1c330a82981d7d27d1202a5cbe3a59581ff67af5e3361f5f90eR49